src/components/filter-box/filter-box.tst.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React, { useState } from "react"
+import { fireEvent } from "@testing-library/react"
+import { DefaultTheme } from "../../theme/default"
+import { testWrapper } from "../../../test-utils"
+import "@testing-library/jest-dom/extend-expect"
+import "jest-styled-components"
+import { FilterBox, FilterBoxProcessing, FilterBoxOption } from "."
+import { Text } from "../typography"
+
+// TODO - figure out TypeError: document.body.createTextRange is not a function
+// one of underlying components might have a problem with jsdom environment lacking the method
+
+export const data = [
+  { name: "Bob", position: "Front End" },
+  { name: "Alice", position: "Front End" },
+  { name: "Chris", position: "Back End" },
+  { name: "May", position: "Back End" },
+]
+
+export const options = [
+  { columnField: "name", type: "text" },
+  { columnField: "position", type: "selection" },
+] as FilterBoxOption[]
+
+const MockedFilterBox = () => {
+  const [employees, setEmployees] = useState(data)
+
+  const handleParse = expressions => {
+    const newData = new FilterBoxProcessing(options).process(data, expressions)
+    setEmployees(newData)
+  }
+
+  return (
+    <>
+      <FilterBox options={options} onParseOk={handleParse} data={employees} />
+      {employees.map(({ name, position }) => (
+        <div key={name}>
+          <Text className="contentClass">{`${name} ${position}`}</Text>
+        </div>
+      ))}
+    </>
+  )
+}
+
+describe("Filter Box test", () => {
+  it(" * should filter results by expression", () => {
```

Comment:
@Azein is the `tst` in filename a typo, or a way to skip tests? :) we can also do `describe.skip("...` or `it.skip("...)`

Reply:
I mentioned it in PR description. Test doesn't work as something inside react-filter-box, maybe the code editor dependency, is breaking in jsdom and it will take time to figure out the solution.